### PR TITLE
Correct the day count in expected hours column in effort details in projects#3212

### PR DESCRIPTION
**Pull request by MohitGururani**
Targets #3212

### Description
-In this task, I successfully resolved an issue related to the calculation of the day's count in the "Expected Hour" column heading. Previously, the calculation was inaccurate as it added a whole day until 10 PM. To fix this, I made the necessary changes to accurately reflect the time period from the start date to the end of the day at 10 PM for each day.
Furthermore, I ensured that the current day is not included in the day's count calculation. Additionally, I extended this calculation to the "Expected Hours Till Today" column heading, providing consistent and accurate information throughout the application.
-In the second task i change the color of 'red arrow icon'.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by MohitGururani** [comment]
**Expected Time**
1.Correcting the expected hour column heading.**2-4 hours**
2.Changing the color of 'red arrow icon'.**2-4 hours**
**Total Expected Time: 8 hours**

**Total Time Taken : 9 hours**

File: resources/js/components/Dashboard/UserDashboardProjects.vue [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+								</span>
+							</div>
+						</div>
+					</div>
+					<hr class="mt-1">
+				</div>
+			</div>
+			<div v-else>
+				<p>Fetching latest data...</p>
+			</div>
```
Comment (GauravGusain98) [inline resources/js/components/Dashboard/UserDashboardProjects.vue:29]: Please correct the identation. The previous indentation was correct, in you current changes it got increased

File: Modules/EffortTracking/Services/EffortTrackingService.php [hunk 2]
```diff
@@ -31,7 +31,6 @@ public function show(array $data, $project)
         $endDate = $project->client->getMonthEndDateAttribute($totalMonths);
         $totalWorkingDays = count($this->getWorkingDays($startDate, $endDate));
         $totalEffort = $project->getCurrentHoursForMonthAttribute($startDate, $endDate);
-        $totalWorkingDays = count($this->getworkingDays($startDate, $endDate));
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Services/EffortTrackingService.php:34]: Why you removed it? It's being used below in the return statement. Please test your code properly

File: Modules/EffortTracking/Services/EffortTrackingService.php [hunk 3]
```diff
@@ -31,7 +31,6 @@ public function show(array $data, $project)
...
         $daysTillToday = count($project->getWorkingDaysList($project->client->month_start_date, $currentDate));
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Services/EffortTrackingService.php:34]: Lets use the same function that we used above `$this->getWorkingDays`

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: Modules/EffortTracking/Services/EffortTrackingService.php (hunk 2 above)
Comment (GauravGusain98) [reply Modules/EffortTracking/Services/EffortTrackingService.php:34]: OK got it was duplicate.

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**